src/utils/firebase/DataFormatting.ts

```diff
@@ -13,3 +13,6 @@ export function snapshotsToValues<T>(
 
   return snapshots.map((snapshot) => snapshotToValue(snapshot));
 }
+export function dataObjectToList<T>(dataObject: Record<string, T>) {
+  return Object.keys(dataObject).map((key) => dataObject[key]);
+}
```

Comment:
Unless I'm missing something, isn't this just homebrew Object.values()? I'd suggest using the standard implementation instead.

Reply:
You are entirely right, I thought for some reason that `Object.values` wouldn't type it properly, but there's no need for this. Will clean up.

Reply:
You forgot that Typescript is the :goat: language with type inference everywhere